### ISSUE TITLE
Cannot access 'GanttEditorOptions' before initialization

### ISSUE THREAD

**Issue by AhsanAyaz** [jsGanttImproved/ng-gantt#15]
I'm facing this issue on an Angular v8 app. I'm using the component in my app.component and have imported the Module too.
```
Uncaught ReferenceError: Cannot access 'GanttEditorOptions' before initialization
    at Module../node_modules/ng-gantt/fesm2015/ng-gantt.js (ng-gantt.js:63)
    at __webpack_require__ (bootstrap:83)
    at Module../src/app/app.module.ts (app.component.ts:13)
    at __webpack_require__ (bootstrap:83)
    at Module../src/main.ts (main.ts:1)
    at __webpack_require__ (bootstrap:83)
    at Object.0 (main.ts:12)
    at __webpack_require__ (bootstrap:83)
    at checkDeferredModules (bootstrap:45)
    at Array.webpackJsonpCallback [as push] (bootstrap:32)
```

Following are the versions I'm at:
```
"jsgantt-improved": "^2.4.4",
"ng-gantt": "^2.4.3",
```

**Comment by mariohmol** [comment]
weird.. I released another vesion.. 2.4.4  please test this out

**Comment by AhsanAyaz** [comment]
@mariohmol , now I'm getting the following error:
```
ERROR in ./src/app/gantt/gantt.component.ts
Module not found: Error: Can't resolve 'ng-gantt/gantt/gantt.editoroptions' in 'src/app/gantt'
```

**Comment by AhsanAyaz** [comment]
Also, I can't import it from `ng-gantt` because it says no exported member for `GanttEditorOptions` for the import below:

```
import { GanttEditorOptions, GanttEditorComponent } from 'ng-gantt';
```

**Comment by mariohmol** [comment]
let me try again here.. sorry.. still using ang7

**Comment by mariohmol** [comment]
weird that travis is passing https://travis-ci.com/jsGanttImproved/ng-gantt

what is your angular versions?

**Comment by mariohmol** [comment]
try v2.4.5
